waspc/examples/todoApp/migrations/20230328075236_add_email/migration.sql

```diff
@@ -0,0 +1,17 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `username` on the `User` table. All the data in the column will be lost.
+  - A unique constraint covering the columns `[email]` on the table `User` will be added. If there are existing duplicate values, this will fail.
+  - Added the required column `email` to the `User` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- DropIndex
+DROP INDEX "User_username_key";
+
+-- AlterTable
+ALTER TABLE "User" DROP COLUMN "username",
+ADD COLUMN     "email" TEXT NOT NULL;
```

Comment:
We can do that 👍 and it makes our existing docs valid for e-mail & password auth for the most part. 

When we tackle the "provider data" issue and make it so that every provider has its own piece of the database, then they would split.